docs/algorithms/kem/kyber.yml

```
@@ -51,6 +57,19 @@ parameter-sets:
     no-secret-dependent-branching-claimed: true
     no-secret-dependent-branching-checked-by-valgrind: true
     large-stack-usage: false
+  - upstream: pqclean-aarch64
+    upstream-id: aarch64
+    supported-platforms:
+    - architecture: ARM64_V8
+      operating_systems:
+      - Linux
+      - Darwin
+      required_flags:
+    common-crypto:
+    - SHA3: liboqs
+    no-secret-dependent-branching-claimed: true
+    no-secret-dependent-branching-checked-by-valgrind: true
```

Are actually checking the aarch64 implementation using Valgrind? I thought that's a TODO based on https://github.com/open-quantum-safe/liboqs/issues/1119.  As such this should (for now) be false.

```suggestion
    no-secret-dependent-branching-checked-by-valgrind: false
```